src/core/lib/security/credentials/plugin/plugin_credentials.cc

```diff
@@ -42,6 +42,16 @@ grpc_plugin_credentials::~grpc_plugin_credentials() {
   }
 }
 
+std::string grpc_plugin_credentials::debug_string() const {
+  if (plugin_.state != nullptr) {
```

Comment:
What's the thinking behind the check that `plugin_.state` is non-null? Presumably a plugin could generate a debug string without needing to refer to its state value (e.g., if there's non-local state that it uses).